docker-compose.yml

```diff
@@ -114,12 +117,21 @@ services:
     environment:
       - HOME=/data
     depends_on:
-      - srly-ose-server
-      - srly-ose-websocket
+      - anthias-server
+      - anthias-websocket
+    networks:
+      default:
+      anthias-extra-1:
+        aliases:
+          - 'anthias-nginx'
+          - 'srly-ose-nginx'
```

Comment:
Why do we need this line? The whole file should be transactional so there should be no `srly-ose-nginx` available after this update.

Reply:
these are only temporary, and will be removed when there's a new build of the webview (given that there's a change in view.cpp)